esy-lib/RewritePrefix.ml

```diff
@@ -0,0 +1,20 @@
+let cmd = ref None
```

Comment:
This seems a little awkward - is there not a way for us to resolve the command location like in `esy-build-package`?

Reply:
Yeah, agree, I'll change it.

Just need to keep in mind that this module will only be usable from `esyCommand.exe` executable as it's going to be resolved against its location.

As I said we need a source code tree reorganisation soon.